api/test/image_integration_test.py

```diff
@@ -125,27 +124,6 @@ def test_oembed_endpoint_for_json():
     assert parsed["license_url"] == "https://creativecommons.org/licenses/by/2.0/"
 
 
-def test_oembed_endpoint_for_xml():
```

Comment:
The [oEmbed spec](https://oembed.com/) allows response to be either JSON or XML so while dropping XML is not a spec violation, it will be a breaking change for the API.

Reply:
I couldn't get the oEmbed endpoint to work :( Has it been working?
I tried this URL:
https://api.openverse.engineering/v1/images/oembed/?format=json&url=https%3A//www.flickr.com/photos/7788419@N05/15218475961
And the same without the `format=json` param, and with `format=xml` - always Cloudflare gateway error 502. 

Is there a redirect in Cloudflare?

Reply:
Yeah, I'm all for dropping these niche endpoint but I also don't want to cause breakages for those who are using them. Data on the usage of this endpoint would be nice.

Reply:
here's some!

<img width="677" alt="CleanShot 2023-03-22 at 14 46 35@2x" src="https://user-images.githubusercontent.com/6351754/227006563-fe1a931b-9363-422f-8fc1-fece5d4b1ca2.png">
<img width="686" alt="CleanShot 2023-03-22 at 14 46 17@2x" src="https://user-images.githubusercontent.com/6351754/227006575-47010c60-8176-4940-a4d7-e4f82fea9ca3.png">

Reply:
I think we can delete it if it's only been used 8 times in the last month.

Reply:
Especially considering at least half of them were probably Olga testing 😁

Reply:
There is context for the removal suggestion in the issue as well. I had the same question as Dhruv and didn't want to remove it if it was being used. We've had several thousand XML requests over the last week, but before that it's basically no one. The usage of the XML is definitely weird for the last week!

Reply:
@sarayourfriend I definitely think your "an api consumer was crawling the ui and following links" theory makes a lot of sense because it would also potentially explain the GET requests to the reporting endpoint.

Reply:
That's exactly what I was thinking too. We don't include the report URL in the result object, but it is in the "additional actions" list on the detail URL's browsable page, so very clickable for a crawler.